### DISCUSSION TITLE
Build ipa for distribution exited with status code 65

### DISCUSSION THREAD

**Discussion by miladdev85** [codemagic-ci-cd/codemagic-docs#1321] [Q&A] [answered]
Check AppName.xcworkspace build settings
`Execute "xcodebuild -workspace /Users/builder/clone/ios/AppName.xcworkspace -scheme AppName -showBuildSettings"`

Archive AppName.xcworkspace
`Execute "xcodebuild -workspace /Users/builder/clone/ios/AppName.xcworkspace -scheme AppName -archivePath build/ios/xcarchive/AppName_zjnxefik.xcarchive archive COMPILER_INDEX_STORE_ENABLE=NO DEVELOPMENT_TEAM=xxyyzz 'CODE_SIGN_IDENTITY=Apple Distribution'"`

Last step before failure:

```
Run script build phase 'Create Symlinks to Header Folders' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'RCT-Folly' from project 'Pods')
Failed to archive /Users/builder/clone/ios/AppName.xcworkspace
```

Build failed :|
Step 8 script Build ipa for distribution exited with status code 65

---

Build ipa for distribution in codemagic.yaml file:
<img width="493" alt="image" src="https://user-images.githubusercontent.com/51476530/212773661-2f38a45e-a1c4-44c9-9bf4-e3eafed954ef.png">

Can someone please help? Why is this happening? What can I do to further debug this?

**Comment by himesh-cm** [comment, marked as answer]
Hi @miladdev85, Can you please share the build id as well, so I can look into the issue further?

**Reply by miladdev85** [reply]
Hi @himesh-cm 
How can I share the build? I'm new to Codemagic so thank you for your patience 😬🙏

**Reply by miladdev85** [reply]
Perhaps build id can help?
Build id: 63c659b5e42bfa55c5b17961

**Reply by himesh-cm** [reply]
Let me check, i will get back to you.

**Reply by miladdev85** [reply]
Thank you so much 🙏

**Reply by himesh-cm** [reply]
Hi @miladdev85,
The error from the logs are `React-Core-AccessibilityResources is automatically signed for development, but a conflicting code signing identity Apple Distribution has been manually specified.`

This is a known issue with Xcode 14.x versions.
Can we please modify the build command below

```
xcode-project build-ipa \
  --workspace "$CM_BUILD_DIR/ios/$XCODE_WORKSPACE" \
  --scheme "$XCODE_SCHEME"
  --archive-flags "-destination 'generic/platform=iOS'" \
  --archive-xcargs "CODE_SIGNING_REQUIRED=YES CODE_SIGNING_ALLOWED=NO"
```

Please re-run the build and let us know how it goes.

**Reply by miladdev85** [reply]
Thanks but now it's failing with exit code 127.

`/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/build_script_8_1_5gq_e5vf: line 6: --archive-flags: command not found`

Build id: `63c6802cb674ee92f64e8c21`

**Reply by miladdev85** [reply]
Woho!

It was the missing `\` after `--scheme "$XCODE_SCHEME"`

Everything is working now. 

Thank you so much for your help and support @himesh-cm 

Maybe it's a good idea to add the part about `--archive-flags "-destination 'generic/platform=iOS'" \
  --archive-xcargs "CODE_SIGNING_REQUIRED=YES CODE_SIGNING_ALLOWED=NO"` in the troubleshooting guide?

**Reply by himesh-cm** [reply]
@miladdev85, Great you were able to figure it out. Let me check regarding the documentation update at our end.
